src/workloads/scale/LLTInsert.yml

```diff
@@ -86,11 +86,11 @@ Actors:
           OperationName: RunCommand
           OperationCommand:
             setParameter: 1
-            transactionLifetimeLimitSeconds: 3600
+            transactionLifetimeLimitSeconds: 10800
```

Comment:
Could you add a trailing comment "# 3 hours" to each of these?

Reply:
sure (and actually with an eye to the future, I will make it slightly longer).